guides/source/active_job_basics.md

```diff
@@ -269,6 +269,12 @@ UserMailer.welcome(@user).deliver_now
 UserMailer.welcome(@user).deliver_later
 ```
 
+You can also configure `ActionMailer::DeliveryJob` to use a specific queue
+adapter, distinct from the one set on `ActiveJob::Base`:
```

Comment:
Isn't this lying? The code seems to be explicitly setting it to `:inline`, and never deferring to the one set on AJ::Base.

(FWIW, I prefer the behaviour described in the comment over [my understanding of] the behaviour in the railtie.)